internals/utils/package.json

```diff
@@ -1,17 +1,134 @@
 {
   "private": true,
   "name": "@vercel-internals/utils",
-  "types": "dist/index.d.ts",
-  "main": "dist/index.js",
+  "types": "dist/internals/utils/src/index.d.ts",
+  "main": "dist/internals/utils/src/index.js",
   "files": [
     "dist"
   ],
   "scripts": {
-    "build": "tsc -p tsconfig.json"
+    "compile-templates": "node scripts/compile-templates.js",
+    "generate-constants": "ts-node scripts/generate-constants.ts",
+    "build": "pnpm generate-constants && pnpm compile-templates && tsc -p tsconfig.json",
+    "test": "jest --env node --verbose --bail tests"
+  },
+  "dependencies": {
+    "@sentry/node": "5.5.0",
+    "@sindresorhus/slugify": "0.11.0",
+    "@tootallnate/once": "1.1.2",
+    "@vercel-internals/types": "*",
```

Comment:
How are you deciding which ones are dependencies vs devDependencies?

Reply:
modules used in `src` are deps
modules used in `tests` (and any `@types`) are devdeps

Reply:
Is `@vercel-internals/types` used in `src`?